packages/api/src/schema/api-schema.yaml

```diff
@@ -2220,12 +2255,6 @@ components:
       type: object
       description: |
         An individual metric about usage of a user.
-      required:
-        - UserID
-        - CreatorID
-        - DeliveryUsageMins
-        - TotalUsageMins
-        - StorageUsageMins
```

Comment:
Are these not always returned?

Reply:
the required items are just for response not request payload and sometimes, if there is data, the api returns and other times it doesn't, which is why i removed the required fields as it really doesn't have impact on user side validation